modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -127,10 +130,38 @@ storage:
           method=manual
 ----
 
+NMstate
+[source, yaml]
+----
+variant: fcos
+version: 1.4.0
+storage:
+  files:
+    - path: /etc/nmstate/ens2.yml
+      mode: 0600
+      contents:
+        inline: |
```

Comment:
I think the original example are expecting a static hostname. So we can state here in nmstate like:

```yml
hostname:
  config: myhost
```